src/main/java/no/finn/unleash/strategy/UserWithIdStrategy.java

```diff
@@ -0,0 +1,33 @@
+package no.finn.unleash.strategy;
+
+import java.util.Map;
+import java.util.Optional;
+
+import no.finn.unleash.UnleashContext;
+
+import static java.util.Arrays.asList;
+
+public final class UserWithIdStrategy implements Strategy {
+
+    protected static final String PARAM = "userIds";
+    private static final String STRATEGY_NAME = "userWithId";
+
+    @Override
+    public String getName() {
+        return STRATEGY_NAME;
+    }
+
+    @Override
+    public boolean isEnabled(Map<String, String> parameters) {
+        return false;
+    }
+
+    @Override
+    public boolean isEnabled(Map<String, String> parameters, UnleashContext unleashContext) {
+        return unleashContext.getUserId()
+                           .map(currentUserId -> Optional.ofNullable(parameters.get(PARAM))
```

Comment:
Almost trivial, but util methods for parsing different parameter types would be nice.